test/admin/Dockerfile

```diff
@@ -0,0 +1,47 @@
+FROM ruby:2.4.0
+
+COPY ./install-freetds.sh .
+RUN ./install-freetds.sh
+
+# throw errors if Gemfile has been modified since Gemfile.lock
+RUN bundle config --global frozen 1
+
+# Update the repositories
+# Install utilities
+# Install XVFB and TinyWM
+# Install fonts
+# Install Python
+RUN apt-get -yqq update && \
+    apt-get -yqq install curl unzip && \
```

Comment:
Is there a reason not to add all the `install` packages over a single `apt-get` command?

Reply:
it's just easier to read IMO.